components/Insurance/Pools/index.tsx

```diff
@@ -77,8 +79,13 @@ const OwnershipCell: React.FC<CProps> = styled(({ pool, className }: CProps) =>
     return (
         <div className={className}>
             <span>
-                {pool.userBalance.toNumber()} {pool.market}
+                {pool.userBalance.toNumber()} {pool.iPoolTokenName} 
             </span>
+            <Link href={pool.iPoolTokenURL}>
+            <StyledTooltip title="View on Etherscan">
```

Comment:
We keep all tooltips separate but this is fine I can handle it

Reply:
@Steven-Yulong-Yan as in the tooltips all live in their own location? I can patch this if you want

Reply:
Yeah there's a tooltip selector where we keep all of them. No worries I'm on it already.